### PR TITLE
Blog app - controllers

### PR DESCRIPTION
**Pull request by Feliverse**
## Description
In this project, I create all controllers and views for the Blog app. As you won't be processing any data yet, you will add simple placeholders for your views with plain HTML.

### Project requirements
-  **Important:** do NOT use `rails generate scaffold` command. The point of this project is to show you how to set up controllers by yourself.
-  Take a look at the wireframes below. Make sure that your app handles the URLs used in the wireframes:  

blog_root_users

blog_user_page

blog_user_all_posts

blog_single_post

-  For each URL you should create:
    - A route.
    - An action in the correct controller.
    - A view file.

-  View files should include only placeholders written in plain HTML, e.g:
    - Here is a list of posts for a given user
routes

-  Make sure that you added code to handle **only** the 4 required URLs.

### Need a big picture? 

Remind me about [the big picture of this project](../sneak_peek.md).

### CODE REVIEW

**Review by CollinsTatang — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README file.

### Please you will need to update your README file to the current template

- [x] Please make sure all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present.
- [x] Please make sure that you add the section **Future Features**, including 1-3 potential features to be added.
- [x] Please remove the Microverse logo from your README file


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class PostsController < ApplicationController
+  def index; end
+end
```
Comment (AnselemOdims) [inline app/controllers/posts_controller.rb]: - [x] Good job working on this controller, but please kindly add the `show` action to handle post show requests

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
+  def index; end
+end
```
Comment (AnselemOdims) [inline app/controllers/users_controller.rb]: - [x] Good job working on this controller, but please kindly add the `show` action to handle user show requests

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Feliverse , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (Feliverse) [reply app/controllers/posts_controller.rb]: Thank to let me know this, I am fixing now

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (Feliverse) [reply app/controllers/users_controller.rb]: Thank to let me know this, I am fixing now

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
```
Comment (danmainah) [inline config/routes.rb]: - [x] [optional] @Feliverse great implementing the changes requested, however, I would recommend you add root route in your routes preferably ` root "users#index"`

**Review by danmainah — APPROVED** [review]
Hi @Feliverse , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ Linter setup correctly and all checks are passing
✔️ Al pages added successfully

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 3 above)
Comment (Feliverse) [reply config/routes.rb]: @danmainah,  Interesting suggestion, I am implementing it now, please let me know if I do it in a correct way

Comment (danmainah) [reply config/routes.rb]: @Feliverse yes its done correctly , great work

**Pull request merged**